src/main/java/HelloWorld.kt

```diff
@@ -0,0 +1,6 @@
+
+fun main(args : Array<String>){
```

Comment:
Diese Datei gehört nicht in `src/main/java` sondern in `src/main/kotlin` - damit das funktioniert, müsst ihr die POM noch ein bisschen erweitern (siehe Vorlesung).